libs/sdk-backend-bear/src/backend/organization/index.ts

```diff
@@ -44,6 +46,10 @@ export class BearOrganization implements IOrganization {
     public styling(): IOrganizationStylingService {
         return new OrganizationStylingService();
     }
+
+    public settings(): IOrganizationSettingsService {
+        return new OrganizationSettingsService();
```

Comment:
This service is not necessary. You can just throw exception here as Bear does not support this service at all.

Alternatively the service can be implemented using Bear API. You can fetch the whole Bear white-labeling settings via https://help.gooddata.com/doc/enterprise/en/expand-your-gooddata-platform/api-reference#operation/getWhitelabelingSettings, update four values that we support for embedded scope white-labeling, and then put updated object back via https://help.gooddata.com/doc/enterprise/en/expand-your-gooddata-platform/api-reference#operation/updateWhiteLabelingSettings. It is more work but SPI will work for Bear as well. However, it will probably not be used so it is only for the sake of completeness of the SPI backends. It's up to you.